queries.sql

```diff
@@ -23,3 +26,81 @@ SELECT * FROM ANIMALS WHERE NAME NOT IN ('Gabumon');
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM ANIMALS WHERE WEIGHT_KG >= 10.4 AND WEIGHT_KG <= 17.3;
+
+
+-------------------------------- DAY 02 --------------------------------
+
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before tranasction.
+BEGIN;
+UPDATE ANIMALS SET SPECIES = 'UNSPECIFIED';
+ROLLBACK;
+
+-- Inside a transaction:
+BEGIN;
+
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+UPDATE ANIMALS SET SPECIES = 'Digimon' WHERE NAME LIKE '%mon';
+
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+UPDATE ANIMALS SET SPECIES = 'Pokemon' WHERE NAME NOT LIKE '%mon';
```

Comment:
**Changes Required**
- [ ] This will set species column to `pokemon` for all animals not ending with `mon`. The requirement is that we should set it to `pokemon` where we don't have species yet. Using `NULL` would be the ideal solution. Let us please make these changes.

Reply:
Got your point, let me fix it quickly